### ISSUE TITLE
S2681 should not report on multiline raw strings

### ISSUE THREAD

**Issue by RalfKoban** [SonarSource/sonar-dotnet#9570]
### Description
Currently, S2681 detects mult-line blocks within raw multi-line strings and reports them to be enclosed in curly braces.

### Expected behavior
As that are only strings and not actual code, S2681 should ignore such strings.

### Actual behavior
S2681 reports a violation within raw multi-line strings.

### Known workarounds
* Deactivate S2681

**Comment by Tim-Pohlmann** [comment]
Hey @RalfKoban, thanks for the report!
I tried this:
```csharp
_ = """
    void If(bool condition)
    {
        if (condition)
            Act.First();
            Act.Second();
    }
    """;
```
and cannot reproduce it. Can you share a reproducer with us? Thanks!

**Comment by RalfKoban** [comment]
Unfortunately, for the moment I cannot share a reproducer as it is production code.

However, the multi-line comment does not contain real C# code. It contains some queries and comments (with `#` ) that contain words such as `interface` or `if` which somehow get interpreted as C# keywords.

**Comment by Tim-Pohlmann** [comment]
Please try to re-create a similar example. Otherwise it is hard for us to understand what's happening.

**Comment by RalfKoban** [comment]
For the moment I cannot re-create an example. It might be that the S2681 violation had been reported by an earlier version of SonarLint and has been addressed in the meanwhile. That's why I close this issue here.
